lib/global.dart

```diff
@@ -94,10 +94,11 @@ class VikunjaGlobalState extends State<VikunjaGlobal> {
     Workmanager().cancelAll().then((value) {
       settingsManager.getWorkmanagerDuration().then((duration) {
         if (duration.inMinutes > 0) {
+          print('Scheduling Periodic task');
           Workmanager().registerPeriodicTask("update-tasks", "update-tasks",
               frequency: duration,
               constraints: Constraints(
-                  networkType: NetworkType.connected, requiresDeviceIdle: true),
```

Comment:
I found in my testing that having the `requiresDeviceIdle` parameter enabled makes the background tasks behave very erratically.